js/index.js

```diff
@@ -65,4 +67,41 @@ messageForm.addEventListener('submit', function(event) {
 
     messageForm.reset();
 
-})
\ No newline at end of file
+})
+
+//Setting up my Project Section
+const username = 'joelezan';
+
+function getGitProjects() {
```

Comment:
Did you mean to use an async function here?  Regular functions return undefined if there is no explicit return.  async functions always return a promise.  A common pattern is to make the function async and then await the fetch.